exploration/open-close-placeholders.md

```diff
@@ -135,9 +137,12 @@ a non-empty sequence of pattern parts (text, placeholders).
 Markup spans may be nested,
 as in `<b>Bold and <i>also italic</i></b>`.
 
-Markup may have localisable options,
+Open and standalone markup may have localisable options,
```

Comment:
I don't think "localizable" and "non-localizable" are valid distinctions. For one thing, our spec won't know the difference (they're all just "options" to us). The localizability is about how the options or values are used.

The intent of the original sentence was to show that there could be language in an option, but this, coupled with lines 143-145 change the meaning somewhat.

What I would do, then, is spend extra time on the topic:

```suggestion
Markup placeholders can contain options.

Some option values might require translation themselves. 
For example, `Click <a title="Link tooltip">here</a> to continue.`

The meaning of markup options is externally defined.
Some uses might require specific values
such as when used by tools to associate markup values
in pairs: `Click {#a id=a1}here{/a ref=a1} to {#a id=a2}continue{/a ref=a2}`.

> [!NOTE]
> Many markup systems do not use or permit options on close values.
```

Reply:
> I don't think "localizable" and "non-localizable" are valid distinctions. 

Agree. That is a piece of registry info, but does not affect anything at this level.

> Many markup systems do not use or permit options on close values.

Note that our markup is limited to only represent what most people call "markup"
(may be this is not a good name, but too late to change, will see what people think).
It can be used to represent spanned strings (iOS, Android), ANSI escapes (command line apps), more.

So I would either not have that "NOTE", or have it, but also add this explanation.
It explains (a bit) why we have something that "traditional markup" does not.

Reply:
> Note that our markup is limited to only represent what most people call "markup"
(may be this is not a good name, but too late to change, will see what people think).

I have tried hard to call it "spannables", but we never incorporated that into the abnf or some of the spec text. Spannable is a better name, IMHO.

Reply:
Sorry, edited the comment above (_"our markup is **NOT** limited to only represent"_)
More readable than correcting in a separate comment, but strikethrough shows the mistake (didn't want to feel like "cheating", changing things "without a trace" :-)